Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
@tius2000 I think this is your issue: https://github.com/SmingHub/Sming/pull/946
So please check your makefile. Problem only with debug output?